.github/workflows/docker-unified.yml

```diff
@@ -472,23 +488,35 @@ jobs:
         if: ${{ needs.setup.outputs.publish != 'true' }}
         with:
           image: ${{ env.DATAHUB_MCE_CONSUMER_IMAGE }}:${{ needs.setup.outputs.unique_tag }}
-      - name: Download MCE Consumer image
+      - name: Download MAE Consumer image
```

Comment:
Is this change correct? Why are we using env `env.DATAHUB_MCE_CONSUMER_IMAG`

Reply:
yes this is correct. The `-` starts a new task. If you see the next lines it is for MAE not MCE